ova/Libraries/provision-opendistro.sh

```diff
@@ -0,0 +1,278 @@
+#!/bin/bash
+char="#"
+debug=''
+sys_type="yum"
+
+logger() {
+
+    echo $1
+}
+
+startService() {
+
+    if [ -n "$(ps -e | egrep ^\ *1\ .*systemd$)" ]; then
+        systemctl daemon-reload 
+        systemctl enable $1.service 
+        systemctl start $1.service 
+        if [ "$?" != 0 ]; then
+            echo "${1^} could not be started."
+            exit 1;
+        else
+            echo "${1^} started"
+        fi
+    elif [ -n "$(ps -e | egrep ^\ *1\ .*init$)" ]; then
+        chkconfig $1 on 
+        service $1 start 
+        /etc/init.d/$1 start 
+        if [ "$?" != 0 ]; then
+            echo "${1^} could not be started."
```

Comment:
Use `logger` function.

Reply:
Done in 53208f3d9e390ee69e4f23799af5b3009001d9c2